<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TiWenchange_Dialog</class>
 <widget class="QDialog" name="TiWenchange_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>381</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="0">
         <widget class="QLabel" name="numberchange_label">
          <property name="text">
           <string>学号</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="sex_label">
          <property name="text">
           <string>体温修改</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>所在房间号</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="sid_lineEdit"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="age_label">
          <property name="text">
           <string>原体温</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="tiwen_lineEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="name_abel">
          <property name="text">
           <string>所在宿舍楼</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="tiwenchange_lineEdit"/>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="stuname_lineEdit"/>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="date_lineEdit"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="college_label">
          <property name="text">
           <string>姓名</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="building_lineEdit"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="number_label">
          <property name="text">
           <string>填报日期</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="room_lineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="update_pushButton">
        <property name="text">
         <string>修改</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
